diff --git a/zig/private/common/translate_c.bzl b/zig/private/common/translate_c.bzl
index 1234567..89abcde 100644
--- a/zig/private/common/translate_c.bzl
+++ b/zig/private/common/translate_c.bzl
@@ -128,7 +128,7 @@ def _builtin_translate_c(*, ctx, zigtoolchaininfo, global_args, compilation_con
         inputs.append(zigtoolchaininfo.zig_lib.file)
 
     ctx.actions.run_shell(
-        command = "${{@}} > {}".format(zig_out.path),
+        command = '"${{@}}" > "{}"'.format(zig_out.path),
         inputs = depset(
             direct = inputs,
             transitive = transitive_inputs,
